catalog/video/video-app/src/main/resources/features.xml

```diff
@@ -28,18 +29,27 @@
     <feature name="mpegts-input-transformer" install="auto" version="${project.version}"
              description="Transform MPEG-TS Files">
         <feature prerequisite="true">video-app</feature>
+        <feature prerequisite="true">security-classification-service</feature>
         <bundle>mvn:org.codice.alliance.video/video-mpegts-transformer/${project.version}
         </bundle>
         <configfile finalname="/etc/DDF_Custom_Mime_Type_Resolver.mpegts.config">
             mvn:org.codice.alliance.video/video-mpegts-transformer/${project.version}/config/mpegts
         </configfile>
     </feature>
 
+    <feature name="security-classification-service" install="auto" version="${project.version}"
```

Comment:
If this security classification service is generic, then the feature shouldn't be defined in the video app.  If it is specific to video, then the security service should move somewhere under catalog/video